src/jarallax.esm.js

```diff
@@ -36,6 +36,7 @@ function getDeviceHeight() {
 let wndW;
 let wndH;
 let wndY;
+let contY;
```

Comment:
_Really not important, but fix if you wish:_ It took me a while to determine `cont` = `container` here. I'd maybe just call it `containerY` even though they abbreviate `window` to `wnd`.

Reply:
Fixed